docs/contribute.rst

```diff
@@ -766,3 +766,24 @@ fix needs to be released before the regular schedule:
 * Tag the commit and publish the release in the same way as for
   regular release
 * Create a pull request with the hotfix release notes changes
+
+
+Releaser
+------------------------------------------------------------------
+
+Taking care of a new ``tmt`` release is not just about performing
+the final steps described above. In this role you should shepherd
+the issues and pull requests like sheep so that they make it to
+the ``main`` branch by the proposed deadline. Here's a couple of
+recommendations which could help you to make the release process
+smooth and timely:
+
+* continually watch the issues & pull requests and gently push
+  them forward if any of them seems to get stuck
+* bring attention especially to those with the high priority, the
+  ``must-have`` issues and pull requests should be finished
```

Comment:
I'm not sure we have a literal `must-have`, the label says `priority | must` and I couldn't find `must-have` in tmt repo.

Reply:
Ah, good point, fixed in e3fc5429.